### ISSUE TITLE
6.0 and Multiplayer (Techtree issue upon enabling the 6.0 alpha)

### ISSUE THREAD

**Issue by Draco18s** [Anuken/Mindustry#2621]
**Platform**: Windows

**Build**: 105

**Issue**: When a player first swaps to the alpha branch and before starting a new campaign, if they join a multiplayer session, they will have all of the tech unlocks that they had available on the non-alpha build.

**Steps to reproduce**: Have tech unlocks (non-alpha), swap to alpha branch, join an alpha multiplayer game

**Expected**: Player should be limited by either their own tech unlocks (and switching to the new system should clear it without having to start a new campaign) or limited by the *host's* unlocks. As the joining player cannot even see the tech tree, limiting by host may make the most sense.

**Comment by Anuken** [comment]
Duplicate of #2573
